constants/CreateFilterConstants.ts

```diff
@@ -1,11 +1,25 @@
-export const marketplaces = [
+export interface Options {
+  value: string
+  label: string
+  premium?: boolean
+}
+
+export const marketplaces: Options[] = [
   { value: 'all', label: 'allMarket' },
   { value: 'Ebay', label: 'Ebay' },
   { value: 'Kleinanzeigen', label: 'Kleinanzeigen' },
   { value: 'AutoScout24', label: 'autoScout' },
   { value: 'Facebook', label: 'facebookMarketplace', premium: true },
   { value: 'Craigslist', label: 'craigslistMarketplace' },
```

Comment:
craiglist cities in the eu have about 50 entries in total with like 1-2 added per day, I don't think it should be selectable